src/ui/pages/Overview/Positions/Positions.tsx

```diff
@@ -814,9 +815,9 @@ export function Positions({
   }
   const moveGasPositionToFront = chainValue !== NetworkSelectValue.All;
   const isSupportedByBackend =
-    chainValue === NetworkSelectValue.All
+    chainValue === NetworkSelectValue.All || !chain
```

Comment:
can we remove `chainValue === NetworkSelectValue.All` here?

Reply:
good catch, look like it!